project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,17 @@
+import { AppRoute, AutorizationStatus } from '../../const';
+import {Navigate} from 'react-router-dom';
+
+type PrivateRouteProps = {
+  authorizationStatus: AutorizationStatus;
+  children: JSX.Element
+}
+
+export default function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Можно сразу достать пропсы, следует в дальнейшем использовать такой подход
```suggestion
export default function PrivateRoute({ authorizationStatus, children }: PrivateRouteProps): JSX.Element {
```